Sources/AWSSDKSwiftCore/AWSClient.swift

```diff
@@ -800,14 +843,14 @@ extension AWSClient.ClientError: CustomStringConvertible {
     }
 }
 
-extension AWSClient {
-    func recordMetrics<Output>(_ future: EventLoopFuture<Output>, service: String, operation: String) -> EventLoopFuture<Output> {
+extension EventLoopFuture {
```

Comment:
While extending ELF makes the calling functions slightly cleaner, I do wonder if it makes sense moving AWSClient metrics recording into an unrelated object.